recoverMissingLumis.py

```diff
@@ -266,13 +266,14 @@ def getFiles(datasetName, runBlacklist, runWhitelist, blockBlacklist,
             for replica in replicaInfo["phedex"]["block"][0]["replica"]:
                 node = replica["node"]
                 cmsSites = siteDB.PNNtoPSN(node)
+                blockLocations.add(node.replace("_MSS","_Disk"))
```

Comment:
I don't remember why I added this line at that day, but I agree it's risky and not correct. We can either skip it if MSS or add to the blockLocations (which AFAIK will be the same as dropped, since we don't map PSN to MSS PNN).